packages/rich-text/README.md

```diff
@@ -14,169 +14,507 @@ _This package assumes that your code will run in an **ES2015+** environment. If
+-   **value** `Object`: Value to change.
+-   **newFormat** `Object`: The new list format object. Choose between `{ type: 'ol' }` and `{ type: 'ul' }`.
 
-### removeFormat
+**Returns**
 
-```js
-removeFormat( value: Object, formatType: string, ?startIndex: number, ?endIndex: number ): Object
-```
+`Object` The changed value.
 
-Remove any format object from a Rich Text value by type from the given `startIndex` to the given `endIndex`. Indices are retrieved from the selection if none are provided.
+### charAt
 
-### toggleFormat
+[src/index.js#L7-L7](src/index.js#L7-L7)
 
-```js
-toggleFormat( value: Object, format: Object ): Object
-```
+Gets the character at the specified index, or returns `undefined` if no
+character was found.
+
+**Parameters**
+
+-   **value** `Object`: Value to get the character from.
+-   **index** `string`: Index to use.
+
+**Returns**
+
+`?string` A one character long string, or undefined.
+
+### concat
+
+[src/index.js#L8-L8](src/index.js#L8-L8)
 
-Toggles a format object to a Rich Text value at the current selection, and returns a new value with the format applied or removed.
+Combine all Rich Text values into one. This is similar to
+`String.prototype.concat`.
+
+**Parameters**
+
+-   **values** `...Object`: Objects to combine.
+
+**Returns**
+
+`Object` A new value combining all given records.
+
+### create
+
+[src/index.js#L9-L9](src/index.js#L9-L9)
+
+Create a RichText value from an `Element` tree (DOM), an HTML string or a
+plain text string, with optionally a `Range` object to set the selection. If
+called without any input, an empty value will be created. If
+`multilineTag` is provided, any content of direct children whose type matches
+`multilineTag` will be separated by two newlines. The optional functions can
+be used to filter out content.
+
+**Parameters**
+
+-   **$1** `?Object`: Optional named argements.
+-   **$1.element** `?Element`: Element to create value from.
+-   **$1.text** `?string`: Text to create value from.
+-   **$1.html** `?string`: HTML to create value from.
+-   **$1.range** `?Range`: Range to create value from.
+-   **$1.multilineTag** `?string`: Multiline tag if the structure is multiline.
+-   **$1.multilineWrapperTags** `?Array`: Tags where lines can be found if nesting is possible.
+
+**Returns**
+
+`Object` A rich text value.
 
 ### getActiveFormat
 
-```js
-getActiveFormat( value: Object, formatType: string ): ?Object
-```
+[src/index.js#L10-L10](src/index.js#L10-L10)
+
+Gets the format object by type at the start of the selection. This can be
+used to get e.g. the URL of a link format at the current selection, but also
+to check if a format is active at the selection. Returns undefined if there
+is no format at the selection.
+
+**Parameters**
+
+-   **value** `Object`: Value to inspect.
+-   **formatType** `string`: Format type to look for.
+
+**Returns**
+
+`?Object` Active format object of the specified type, or undefined.
+
+### getSelectionEnd
```

Comment:
`value.start` and `value.end` are going to stay, so I guess these are a bit useless. Maybe we should remove them? They were never documented before.

Reply:
Not sure I follow how this comment relates to `getSelectionEnd` :) FWIW, I'm barely familiar with the `rich-text` package.